static/scss/resource-picker-dialog.scss

```diff
@@ -56,7 +56,7 @@
 
 .resource-picker-dialog {
   .modal-dialog {
-    max-width: 650px;
+    max-width: fit-content;
```

Comment:
What's the goal of this? Is it just to make more room for the tabs? I think this may have some undesirable affects:

<img width="2560" alt="Screen Shot 2022-04-25 at 1 05 02 PM" src="https://user-images.githubusercontent.com/9010790/165138692-b90cf153-2a0d-4db6-bdd1-bd55afc9035c.png">

One option would be to just leave this change out, and improve the tab stuff a little lated. Related: https://github.com/mitodl/ocw-hugo-projects/pull/161#discussion_r856499430

Reply:
Yes this change was done yo make more room for the video gallery tab. I have removed it for now. 
PS, I wasn't aware there was already a thread for this change